apps/web/src/pages/templates/TemplatesListNoData.tsx

```diff
@@ -73,7 +75,10 @@ export const TemplatesListNoData = ({
 
   return (
     <NoDataHolder data-test-id="no-workflow-templates-placeholder">
-      <NoDataSubHeading>Start from a blank workflow or use a template</NoDataSubHeading>
+      <NoDataSubHeading className={css({ color: 'legacy.success !important' })}>
+        Start from a blank workflow or use a template
+      </NoDataSubHeading>
+      <Test />
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** We will remove this before merging! This is just to validate that everything is working across the two‏